packages/web/README.md

```diff
@@ -92,7 +92,7 @@ const UrgeWithPleasureComponent = () => (
     isPlaying
     duration={60}
     initialRemainingTime={15}
-    colors={[['#A30000']]}
+    colors={[['#A30000'],0]}
```

Comment:
This one is the same as above and the color transition duration is placed in a wrong place, it should go as ` colors={[['#A30000', 1]]}`